nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -0,0 +1,445 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List
+from typing import Type
+
+from nncf.common.graph.operator_metatypes import INPUT_NOOP_METATYPES
+from nncf.common.graph.operator_metatypes import OUTPUT_NOOP_METATYPES
+from nncf.common.graph.operator_metatypes import OperatorMetatype
+from nncf.common.graph.operator_metatypes import OperatorMetatypeRegistry
+from nncf.common.hardware.opset import HWConfigOpName
+
+OV_OPERATION_METATYPES = OperatorMetatypeRegistry('openvino_operator_metatypes')
+
+
+class OVOpMetatype(OperatorMetatype):
+    op_names = []  # type: List[str]
+
+    @classmethod
+    def get_all_aliases(cls) -> List[str]:
+        return cls.op_names
+
+
+@OV_OPERATION_METATYPES.register()
+class OVConvolutionMetatype(OVOpMetatype):
+    name = 'ConvOp'
+    op_names = ['Convolution']
+    hw_config_names = [HWConfigOpName.CONVOLUTION]
+
+
+@OV_OPERATION_METATYPES.register()
+class OVConvolutionBackpropDataMetatype(OVOpMetatype):
+    name = 'ConvTransposeOp'
+    op_names = ['ConvolutionBackpropData']
+    hw_config_names = [HWConfigOpName.CONVOLUTION]
+
+
+@OV_OPERATION_METATYPES.register()
+class OVReluMetatype(OVOpMetatype):
+    name = 'ReluOp'
+    op_names = ['ReLU']
+
+
+@OV_OPERATION_METATYPES.register()
+class OVGeluMetatype(OVOpMetatype):
+    name = 'GeluOp'
+    op_names = ['Gelu']
+    hw_config_names = [HWConfigOpName.GELU]
+
+
+@OV_OPERATION_METATYPES.register()
+class OVEluMetatype(OVOpMetatype):
+    name = 'EluOp'
+    op_names = ['Elu']
+
+
+@OV_OPERATION_METATYPES.register()
+class OVPReluMetatype(OVOpMetatype):
+    name = 'PReluOp'
+    op_names = ['PReLU']
+
+
+@OV_OPERATION_METATYPES.register()
+class OVSigmoidMetatype(OVOpMetatype):
+    name = 'SigmoidOp'
+    op_names = ['Sigmoid']
+
+
+@OV_OPERATION_METATYPES.register()
+class OVHardSigmoidMetatype(OVOpMetatype):
+    name = 'HardSigmoidOp'
+    op_names = ['HardSigmoid']
+
+
+@OV_OPERATION_METATYPES.register()
+class OVAveragePoolMetatype(OVOpMetatype):
+    name = 'AveragePoolOp'
+    op_names = ['AvgPool']
+    hw_config_names = [HWConfigOpName.AVGPOOL]
+
+
+@OV_OPERATION_METATYPES.register()
+class OVMaxPoolMetatype(OVOpMetatype):
+    name = 'MaxPoolOp'
+    op_names = ['MaxPool']
+    hw_config_names = [HWConfigOpName.MAXPOOL]
+
+
+@OV_OPERATION_METATYPES.register()
+class OVConstantMetatype(OVOpMetatype):
+    name = 'ConstantOp'
+    op_names = ['Constant']
```

Comment:
Is the `Constant` metatype will be useful since `NNCFGraph` have no constants?

Reply:
The `NNCFGraph` for OV contains constants.